packages/lint-contrast/README.md

```
@@ -0,0 +1,130 @@
+lint-contrast --project /path/to/project --glob "src/**/*.tsx" --css src/index.css
+
+# JSON 形式で出力
+lint-contrast --project . --format json
+
+# unknown を error に昇格 (CI で厳格にする場合)
+lint-contrast --project . --warn-as-error
+```
+
+### CLI オプション
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)